collectors/ebpf.plugin/README.md

```diff
@@ -237,6 +334,12 @@ receiving data for connections when the `UDP` protocol is used.
 Like the previous chart, this one also monitors `udp_sendmsg` and `udp_recvmsg`, but instead of showing the number of
 calls, it monitors the number of bytes sent and received.
 
+### Apps
+
+These are tracepoints related to OOM killing processes, netdata uses one of them to show when these events heppen:
```

Comment:
```suggestion
These are tracepoints related to OOM killing processes, netdata uses one of them to show when these events happen:
```
or, really, we should just remove everything after the comma. Why would we re-explain what the tracepoint does?
```suggestion
These are tracepoints related to OOM killing processes.
```

Reply:
Agree with excluding everything after comma. But if we don't, capitalize Netdata.